src/services/completions.ts

```diff
@@ -1505,6 +1506,7 @@ namespace ts.Completions {
             }
 
             const variableDeclaration = getVariableDeclaration(location);
+            const isFileSourceAModule = !!location.getSourceFile().externalModuleIndicator;

```

Comment:
`sourceFile` is already in scope, so you can just do `sourceFile.externalModuleIndicator` inline in the condition.

Reply:
Done.